python-sdk/tests/files/locations/test_wasb.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
```

Comment:
We should add more tests and integration test for various scenarios for loading a file and folder from Azure blob storage

Reply:
@sunank200 the OpenLineage function since is repeated in other locations, would it make sense to have it in the location base class?
We will add the size method and test it.

Reply:
The name and namespace for openlineage can be in different formats. That's the reason that its overridden